vms/platformvm/txs/executor/advance_time_test.go

```diff
@@ -814,35 +814,6 @@ func TestAdvanceTimeTxDelegatorStakers(t *testing.T) {
 	require.Equal(env.config.MinDelegatorStake+env.config.MinValidatorStake, vdrWeight)
 }
 
-// Test method InitiallyPrefersCommit
-func TestAdvanceTimeTxInitiallyPrefersCommit(t *testing.T) {
```

Comment:
q: why simply removing it? Shouldn't we update it so that, calling options on the block, the commit is the prefered one?

Reply:
This test was removed because the functionality was removed. The `txs` package no longer does anything w.r.t. option voting. I think the added tests in `vms/platformvm/block/executor/block_test.go` should now cover what this was previously testing.